docs/core/compatibility/core-libraries/7.0/reflection-invoke-exceptions.md

```diff
@@ -0,0 +1,68 @@
+---
+title: ".NET 7 breaking change: Reflection invoke API exception changes"
+description: Learn about the .NET 7 breaking change in core .NET libraries where the exceptions thrown by reflection invoke APIs have changed.
+ms.date: 04/29/2022
+---
+# Changes to reflection invoke API exceptions
+
+The exceptions thrown when calling reflection invoke APIs have changed.
+
+## Previous behavior
+
+- Previously, when an invoked method that [returns a value by reference](../../../../csharp/programming-guide/classes-and-structs/ref-returns.md) returned `null`, a <xref:System.NullReferenceException> was thrown.
+
+- For constructors, the following exceptions were thrown:
+
+  - Transient exceptions, including <xref:System.OutOfMemoryException>.
+  - An <xref:System.OverflowException> when a negative value was passed for the length parameter of an array.
+
+- When `null` was passed for a [byref-like](xref:System.Type.IsByRefLike) parameter without the `ref` modifier (that is, passed by value), no exception was thrown, and the runtime substituted a default value for the null value.
```

Comment:
For "byref-like", is this an appropriate place to link to? https://docs.microsoft.com/en-us/dotnet/api/system.type.isbyreflike?view=net-6.0